addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -56,17 +56,54 @@ var RatingPopupComposer = publicWidget.Widget.extend({
 
 publicWidget.registry.RatingPopupComposer = publicWidget.Widget.extend({
     selector: '.o_rating_popup_composer',
+    custom_events: {
+        reload_rating_popup_composer: '_onReloadRatingPopupComposer',
+    },
 
     /**
      * @override
      */
     start: function () {
-        var ratingPopupData = this.$el.data();
-        var ratingPopup = new RatingPopupComposer(this, ratingPopupData);
+        this.ratingPopupData = this.$el.data();
+        this.ratingPopupData.display_composer = !(this.ratingPopupData.disable_composer) || !session.is_website_user;
```

Comment:
Seems like these parentheses are misplaced. This will display the composer if it's not disabled in the options, or if the user is not logged in. The second part is not correct, it's just not visible because in our current use cases, we compute disable_composer with slide_channel.can_review which will return false when the user is not logged in, but it's not guaranteed that this will remain the case in the future.

What I suggested was `!(this.ratingPopupData.disable_composer || !session.is_website_user)` ie, if the composer is not disabled and the user is logged in. I guess the most readable way would be `!this.ratingPopupData.disable_composer && session.is_website_user`

Reply:
Done.

Reply:
Seems like you made a mistake and kept the || instead of the &&